README.md

```diff
@@ -281,23 +291,22 @@ view: headquater_building_floors {
     
 }
 
-
 ```
 
 ##### Explore file:
 
 ```LookML
-include: "restaurant_chain.view"
+include: "restaurant_chain.view.lkml"
    
-explore: chains_table {
-  view_name: chains_table
-  from: chains_table
-  label: "chains_table explore"
-  description: "chains_table explore"
+explore: JSON_TABLE {
```

Comment:
Can this view in the example be created with table name `chains_table` to keep the diff to a minimum? It allows to easier review / see the outcome and impact of the changes.

Reply:
Sure I'll make the changes!